curriculum/challenges/english/25-front-end-development/quiz-javascript-higher-order-functions/66edcdd18a4ef8df16e6bb7e.md

```diff
@@ -17,439 +17,488 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Consider the following statements about JavaScript higher-order functions. Which one presents a common misconception?
 
 #### --distractors--
 
-Placeholder distractor 1
+Higher-order functions can greatly enhance code readability and maintainability by enabling functional programming techniques.
 
 ---
 
-Placeholder distractor 2
+Higher-order functions like map, filter, and reduce are powerful tools for array manipulation, but they are not unique to functional programming.
 
 ---
 
-Placeholder distractor 3
+Higher-order functions may introduce complexity in understanding code, but they can also lead to more expressive and concise solutions.
 
 #### --answer--
 
-Placeholder answer
+All functions in JavaScript, including those that do not manipulate or return other functions, can be classified as higher-order functions.
```

Comment:
```suggestion
All functions in JavaScript, including those that do not take or return other functions, can be classified as higher-order functions.
```
Not sure what is meant by "manipulate" here, but I assume we are talking about taking a callback?